src/main/java/io/jenkins/plugins/synopsys/security/scan/extension/pipeline/SecurityScanStep.java

```diff
@@ -229,7 +229,7 @@ public void setBlackduck_scan_failure_severities(String blackduck_scan_failure_s
 
     @DataBoundSetter
     public void setBlackduck_automation_prcomment(Boolean blackduck_automation_prcomment) {
-        this.blackduck_automation_prcomment = blackduck_automation_prcomment ? true : null;
+        this.blackduck_automation_prcomment = blackduck_automation_prcomment ? true : false;
```

Comment:
Same here.

Updating to this should work fine -
`this.blackduck_automation_prcomment = blackduck_automation_prcomment`